contnumber.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open('C://22BCA192//PYTHON//contact1.csv','w',newline='') as obj:
+    p=csv.writer(obj)
+    h=['name','contact']
+    p.writerow(h)
+    data=[['om',1234567890],
+          ['sai',1223456789],
+          ['ram',1234455777],
+         ['radha',1234455777],
+         ['gopal',345678902]]
+    p.writerows(data)

```

Comment:
writerows()? why you can not write writerow()?

Reply:
writerow() can take one line and writerows() take multiple  line for writting.